AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -13,7 +13,10 @@
 from typing import Callable, Tuple
 
 import azlmbr
-import azlmbr.legacy.general as general
+try:
+    import azlmbr.atomtools.general as general  # Standard MaterialEditor or similar executable test.
+except ModuleNotFoundError:  # Editor test.
```

Comment:
This comment could probably be similar to `azlmbr.atomtools is not yet available in the Editor`

Reply:
This will be fixed in the next update I'm going to push.